lib/bookingsync_portal.rb

```diff
@@ -60,6 +60,9 @@ module BookingsyncPortal
   # message bus channel scope
   mattr_accessor :message_bus_channel_scope
 
+  mattr_accessor :rentals_synchronizer
```

Comment:
a line of explanation in a comment above this mattr_accessor definition would be welcome - to have some more clarity and also to keep it consistent with the rest of the file